### PR TITLE
Fix code formatting in Nav block view file

### PR DESCRIPTION
**Pull request by getdave**
## What?

Fixes code formatting as identified in https://github.com/WordPress/gutenberg/pull/60157#discussion_r1537322587

## Why?

- Formatting. 
- Accuracy of comments.

## How?

Code

## Testing Instructions

Run testing steps from https://github.com/WordPress/gutenberg/pull/60130.

### CODE REVIEW

**Comment by getdave** [comment]
Once merged let's bring this into https://github.com/WordPress/gutenberg/pull/60157

File: packages/block-library/src/navigation/view.js [hunk 1]
```diff
@@ -62,18 +62,20 @@ const { state, actions } = store(
...
+				) {
 					actions.openMenu( 'hover' );
+				}
 			},
 			closeMenuOnHover() {
 				const { type, overlayOpenedBy } = getContext();
 				if (
 					type === 'submenu' &&
-					// Only open on hover if the overlay is closed.
+					// Only close on hover if the overlay is closed.
```
Comment (getdave) [inline packages/block-library/src/navigation/view.js:73]: I'm confused as to why we close on hover when something is closed...

Comment (draganescu) [reply packages/block-library/src/navigation/view.js:73]: Because if overlay is open the submenu should not be closed as it's always expanded.

**Review by draganescu — APPROVED** [review]

**Comment by bph** [comment]
I just cherry-picked this PR to the release/18.0 branch to get it included in the next release: fc77423bd0

**Comment by getdave** [comment]
I just cherry picked this into WP 6.5 RC 4 branch https://github.com/WordPress/gutenberg/pull/60241/commits/13a77c794a79e959e78fb021c1a2fd6ccfaf6bd5

**Pull request merged**